### PR TITLE
tuna/example/example_tables.py - type hints 

### PR DESCRIPTION
**Pull request by technicalgrp89**
Add type hints to exsisting code
remove unused code
test against the code changes mypy std's
pass pylint checks
pass ypaf std;s
Pass the CI tests

### CODE REVIEW

File: tuna/example/example_tables.py [hunk 1]
```diff
@@ -41,19 +43,21 @@ class JobEnum(enum.Enum):
...
+  running: int = 3
+  completed: int = 4
+  error: int = 5
 
 
 class Job(BASE):
   """Represents class for job table"""
-  __tablename__ = "job"
-  __table_args__ = (UniqueConstraint('reason', 'session', name="uq_idx"),)
+  __tablename__: Table = "job"
```
Comment (msaudulhassan) [inline tuna/example/example_tables.py]: Seems to be a `str`

File: tuna/example/example_tables.py [hunk 2]
```diff
@@ -41,19 +43,21 @@ class JobEnum(enum.Enum):
...
+  __table_args__: Tuple[str] = (UniqueConstraint('reason',
```
Comment (msaudulhassan) [inline tuna/example/example_tables.py]: wouldn't `Tuple[Any]` be better?

File: tuna/example/example_tables.py [hunk 3]
```diff
@@ -65,9 +69,9 @@ def session(self):
   machine_id = Column(Integer, nullable=False, server_default="-1")
 
 
-def get_tables():
+def get_tables() -> List[str]:
   """Returns a list of all Example lib DB tables"""
-  tables = []
+  tables: List[str] = []
```
Comment (msaudulhassan) [inline tuna/example/example_tables.py]: `List[BASE]` perhaps?

**Review by msaudulhassan — CHANGES_REQUESTED** [review]

File: tuna/example/example_tables.py [hunk 4]
```diff
@@ -41,19 +43,21 @@ class JobEnum(enum.Enum):
...
   # pylint: disable=invalid-name ; names represent entries in job_enum column
   # pylint: disable=duplicate-code
-  new = 1
-  running = 3
-  completed = 4
-  error = 5
+  new: int = 1
+  running: int = 3
+  completed: int = 4
+  error: int = 5
```
Comment (cderb) [inline tuna/example/example_tables.py]: Definition for an Enum would by necessity be an int. If mypy doesn't complain, I don't think we need the int hints here.

**Review by msaudulhassan — APPROVED** [review]

File: tuna/example/example_tables.py [hunk 5]
```diff
@@ -65,9 +68,9 @@ def session(self):
   machine_id = Column(Integer, nullable=False, server_default="-1")
 
 
-def get_tables():
+def get_tables() -> List[str]:
```
Comment (cderb) [inline tuna/example/example_tables.py]: List[BASE]

**Review by cderb — CHANGES_REQUESTED** [review]

File: tuna/example/example_tables.py [hunk 6]
```diff
@@ -49,11 +50,13 @@ class JobEnum(enum.Enum):
 
 class Job(BASE):
   """Represents class for job table"""
-  __tablename__ = "job"
-  __table_args__ = (UniqueConstraint('reason', 'session', name="uq_idx"),)
+  __tablename__: str = "job"
+  __table_args__: Tuple[UniqueConstraint] = (UniqueConstraint('reason',
```
Comment (cderb) [inline tuna/example/example_tables.py:54]: I'm not sure what the right typing for this is, but something seems off.

Comment (msaudulhassan) [reply tuna/example/example_tables.py:54]: @cderb I'd recommend https://github.com/ROCmSoftwarePlatform/MITuna/pull/912#discussion_r1224266350

**Review by cderb — APPROVED** [review]

**Pull request merged**